scripts/openstack-quickstart-demosetup

```diff
@@ -264,7 +264,8 @@ EODASHDB
 SESSION_COOKIE_SECURE = CSRF_COOKIE_SECURE = USE_SSL
 EOSEC
     # sync dashboard DB "after" the database is created
-    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' syncdb --noinput"
+    run_as wwwrun "cd /srv/www/openstack-dashboard; umask 0027; python -m 'manage' makemigrations"
```

Comment:
> You say that the migrations are not distributed, in that case makemigration is indeed needed.

No, Im saying that horizon has no migrations and will not have as its database-free and it tries to keep it that way. Same for django_openstack_auth :)

And for sessions, we only need the migrate command as stated on the docs: https://docs.djangoproject.com/en/1.10/topics/http/sessions/#using-database-backed-sessions